functions/__fisher_plugin_disable.fish

```diff
@@ -1,5 +1,5 @@
 function __fisher_plugin_disable -a plugin path
-    __fisher_plugin_walk "$plugin" "$path" | while read -l class source target name
+    __fisher_plugin_walk "$plugin" "$path" | while command read -l class source target name
```

Comment:
`read` is a builtin, so it would be `builtin read`, however, let's stick to the most basic commands for now.